utils.js

```diff
@@ -217,20 +217,42 @@ export function getLicenses(pkg, format = "xml") {
             licenseContent.id = l;
             licenseContent.url = "https://opensource.org/licenses/" + l;
           } else if (l.startsWith("http")) {
-            if (!l.includes("opensource.org")) {
-              licenseContent.name = "CUSTOM";
-            } else {
+            if (
+              l.includes("mit-license") ||
+              l.includes("://github.com/dotnet/standard/") ||
+              l.includes("://github.com/dotnet/corefx/") ||
+              l.includes("://github.com/dotnet/core-setup/") ||
+              l.endsWith("?linkid=869050") // https://github.com/mono/mono/blob/main/LICENSE
+            ) {
+              licenseContent.id = "MIT";
+            } else if (
+              l.endsWith("?LinkId=329770") || // https://dotnet.microsoft.com/en-us/dotnet_library_license.htm
```

Comment:
Could we try using the known-licenses.json? If not, the full URL instead of a suffix since this might inadvertently match other URLs.

Reply:
Thanks for the hint, way better. Will take care next week.

Reply:
Thank you. The improvements look so good so can't wait!